test/MeshData_tests.jl

```diff
@@ -274,4 +274,84 @@ approx_elem_types_to_test = [(Polynomial(), Hex()),
         @test uf ≈ uf[mapP] 
         
     end
+    @testset "TensorProductWedge MeshData" begin
+        element_type = Wedge()
+        tol = 5e2*eps()
+        @testset "Degree $tri_grad triangle" for tri_grad = [2, 3, 4, 5]
+            @testset "Degree $line_grad line" for line_grad = [2, 3, 4, 5]
```

Comment:
```suggestion
        @testset "Degree $tri_grad triangle" for tri_grad = [2, 3]
            @testset "Degree $line_grad line" for line_grad = [2, 3]
```
I think we can just test tri_grad, line_grad = [2, 3] so that we don't have 16 test cases 😅.